src/Mews.Fiscalization.Greece.Tests/IntegrationTests/AadeClientTests.cs

```diff
@@ -34,13 +35,13 @@ public async Task CheckUserCredentials()
 
         [Theory(Skip = "Temporary skip")]
         [MemberData(nameof(AadeTestInvoicesData.GetInvoices), MemberType = typeof(AadeTestInvoicesData))]
-        public async Task ValidInvoiceDocumentWorks(InvoiceDocument invoiceDoc)
+        public async Task ValidInvoiceDocumentWorks(ISequentialEnumerable<Invoice> invoices)
```

Comment:
```suggestion
        public async Task ValidInvoicesWork(ISequentialEnumerable<Invoice> invoices)
```

Just to keep the naming consistentn.